solutions/haskell/01-ns2/code/your_program.sh

```
@@ -0,0 +1,24 @@
+# - Edit this to change how your program compiles locally
+# - Edit .codecrafters/compile.sh to change how your program compiles remotely
+(
+  cd "$(dirname "$0")" # Ensure compile steps are run within the repository directory
+  stack build
+)
+
+# Copied from .codecrafters/run.sh
+#
+# - Edit this to change how your program runs locally
+# - Edit .codecrafters/run.sh to change how your program runs remotely
+exec $(stack path --local-install-root)/bin/codecrafters-bittorrent-exe "$@"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)